pkg/helm/template_test.go

```diff
@@ -0,0 +1,34 @@
+package helm
+
+import (
+	"github.com/golangplus/testing/assert"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chartutil"
+	kubefake "helm.sh/helm/v3/pkg/kube/fake"
+	"helm.sh/helm/v3/pkg/storage"
+	"helm.sh/helm/v3/pkg/storage/driver"
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+func TestRenderManifests(t *testing.T) {
+
+	chart := "https://srv-file5.gofile.io/download/dtv19k/influxdb-3.1.1.tgz"
+	store := storage.Init(driver.NewMemory())
+	actionConfig := &action.Configuration{
+		Releases:     store,
+		KubeClient:   &kubefake.PrintingKubeClient{Out: ioutil.Discard},
+		Capabilities: chartutil.DefaultCapabilities,
+		Log:          func(format string, v ...interface{}) {},
+	}
+	rel, err := RenderManifests("test", chart, actionConfig)
+	if err != nil {
+		t.Error(err)
+	}
+	txt, ok := rel.(string)
+	if !ok {
+		t.Error("Failed to get template")
+	}
+	assert.True(t, "", strings.HasPrefix(txt, "---"))
```

Comment:
We should test if we got yaml files that correspond to the input chart or make a real unit test out of it.